core-bundle/tests/Image/Studio/FigureBuilderTest.php

```diff
@@ -693,10 +696,14 @@ public function provideLightboxResourcesOrUrls(): \Generator
             $relativeFilePathWithInvalidExtension, [null, $relativeFilePathWithInvalidExtension], false,
         ];
 
-        yield 'external url/path with valid extension' => [
+        yield 'external url/path with valid extension (1)' => [
```

Comment:
I generally do not like the `(1)` and `(2)` suffixes. The titles should say "external url/path with a valid lowercase extension" and "external url/path with a valid uppercase extension", so when a test fails, you do not have to look up the source code to see what `(1)` and `(2)` refers to.

Reply:
see 0f44d5452060f5dca2362af76fa3b525311a1d11